test/integration/Highlight.e2e.test.js

```diff
@@ -52,4 +52,41 @@ describe('Highlights', () => {
         // Assert highlight creation mode is not active
         cy.getByTestId('bp-AnnotationsControls-highlightBtn').should('not.have.class', 'is-active');
     });
+
+    it('should restrict highlight annotation to single page', () => {
+        // Show the preview
+        cy.showPreview(Cypress.env('FILE_ID_DOC'));
+
+        // Wait for the empty highlight layer to be present
+        cy.getByTestId('ba-Layer--highlight');
+
+        let firstPageEl;
+
+        // Select texts across pages
+        cy.get('.textLayer')
+            .first()
+            .children()
+            .last()
+            .trigger('mousedown')
+            .then($firstPageEl => {
+                firstPageEl = $firstPageEl[0];
```

Comment:
Is this an actual "page" or is it a text layer? Should we select on the pages, then dive into the text layer of the page(s) we want?